src/index.js

```diff
@@ -201,7 +202,7 @@ export const Table = ({
       )}
       {visibleColumns.length > 0 ? (
         <TableContainer>
-          <MUITable {...getTableProps(tableProps)}>
+          <MUITable style={{ tableLayout: 'fixed' }} {...getTableProps(tableProps)}>
```

Comment:
consider using css-in-js instead of inline `style` prop (see [this](https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad) for reasons)